spec/policies/membership_application_policy_spec.rb

```diff
@@ -0,0 +1,136 @@
+require 'rails_helper'
+
+
+describe MembershipApplicationPolicy do
+
+  shared_examples_for 'cannot change or destroy status' do
+    describe 'cannot edit, update, or destroy status' do
+      subject { described_class.new(this_user, this_application) }
+
+      it 'cannot edit the status' do
+        is_expected.to forbid_mass_assignment_of(:status).for_action(:edit)
+      end
+      it 'cannot update the status' do
+        is_expected.to forbid_mass_assignment_of(:status).for_action(:update)
+      end
+      it 'cannot destroy the status' do
+        is_expected.to forbid_mass_assignment_of(:status).for_action(:destroy)
+      end
+
+    end
+  end
+
+  shared_examples_for 'can create status' do
+    describe 'can create status' do
+      subject { described_class.new(this_user, this_application) }
+      it 'can create a status' do
+        is_expected.to permit_mass_assignment_of(:status).for_action(:create)
+      end
+    end
+  end
+
+  shared_examples_for 'can see the status' do
+    describe 'can show status' do
+      subject { described_class.new(this_user, this_application) }
+      it 'can show the status' do
+        is_expected.to permit_mass_assignment_of(:status).for_action(:show)
+      end
+    end
+  end
+
+
+  describe 'policy for the status attribute' do
+
+    let(:application_owner) { create(:user, email: 'user_1@random.com') }
+    let(:admin) { create(:user, email: 'admin@sgf.com', admin: true) }
+    let(:not_the_owner) { create(:user, email: 'user_2@random.com') }
+
+    let(:application) { create(:membership_application,
+                               user: application_owner) }
+
+
+    describe 'For the MembershipApplication creator' do
+      it_behaves_like 'can see the status' do
+        let(:this_application) { application }
+        let(:this_user) { application_owner }
+      end
```

Comment:
This uses the shared example to test that the application owner can see the 'status' attribute.  Given the application, it tests that if the user is the application owner, the user (application owner) can see the _status_ parameter, according to the policy.
 
This it block (and others) uses these `let` statements to set the variables that are used in the shared example.  

Is that what you're asking?